### DISCUSSION TITLE
Export trip results on end of a trip

### DISCUSSION THREAD

**Discussion by deregtervictor** [OSTA-group/osta#54] [Ideas]
After you finish a trip it would be cool if you could export something that visually represents your trip.

**Comment by deregtervictor** [comment]
In the 'looks like you're done' popup put a nice overview of the trip and let the user be able to export or end the trip.

**Reply by deregtervictor** [reply]
![image](https://github.com/OSTA-group/osta/assets/101172722/f2408b84-1c24-4cee-81d5-1c64ebe68254)

**Reply by insuT0ver** [reply]
I would place a map view with the landmarks in the center.

**Reply by deregtervictor** [reply]
Indeed! Would be cool, I'm on to it.

**Comment by BauwenDR** [comment]
Update on the implementation: The popup has been implemented with a map view showing the trip. The share button has not been fully implemented yet, we still need to research how to take a screenshot of the map so users can share it.
